roles/idrac_os_deployment/vars/main.yml

```
@@ -56,6 +57,7 @@ idrac_os_deployment_custom_iso_file: ""
 idrac_os_deployment_hybrid_cmd: isohybrid --uefi %s
 idrac_os_deployment_checksum_cmd: implantisomd5 %s
 idrac_os_deployment_xorriso_cmd: "xorriso -osirrox on -indev %s -extract / %s"
+idrac_os_deployment_xorriso_ubuntu_cmd: "7z -y x %s -o%s"
```

```suggestion
idrac_os_deployment_xorriso_7z_cmd: "7z -y x %s -o%s"
```
Also, update the reference as applicable.

---

Lets also document the use of 7z in [README.md](https://github.com/Sakethanne/dellemc-openmanage-ansible-modules/blob/ubuntu_support/roles/idrac_os_deployment/README.md) under `development and `production` library requirements.

---

I also reccomend not to call variable `idrac_os_deployment_xorriso_7z_cmd` since it is not using `xorriso`